authorization-service/src/functions/basicAuthorizer/handler.ts

```diff
@@ -0,0 +1,40 @@
+import 'source-map-support/register';
+
+const generatePolicy = (principalId, effect, resource) => {
+  const authResponse: any = {};
+
+  authResponse.principalId = principalId;
+  if (effect && resource) {
+    const policyDocument: any = {};
+    policyDocument.Version = '2012-10-17';
+    policyDocument.Statement = [];
+    const statementOne: any = {};
+    statementOne.Action = 'execute-api:Invoke';
+    statementOne.Effect = effect;
+    statementOne.Resource = resource;
+    policyDocument.Statement[0] = statementOne;
+    authResponse.policyDocument = policyDocument;
+  }
+
+  return authResponse;
+}
+
+const deny = (methodArn) => generatePolicy('user', 'Deny', methodArn);
+const allow = (methodArn) => generatePolicy('user', 'Allow', methodArn);
+
+export const main = async (event, _, callback) => {
```

Comment:
btw, there is a typing for this handler, called APIGatewayTokenAuthorizerHandler. It should be in `aws-lambda` package